File: DS_P1_S3/C++/Prestamo.cpp
#include "Prestamo.h"

float Prestamo::getImporte() {
	return this->importe;
}

int Prestamo::getNum_cuotas() {
	return this->num_cuotas;
}

float Prestamo::getInteres() {
	return this->interes;
}

Prestamo::Prestamo() {
	importe = 0;
	num_cuotas = 0;
	interes = 0;
}

Prestamo::Prestamo(float imp, int cuotas, float interes){
    importe = imp;
    num_cuotas = cuotas;
    this->interes = interes;
}

Prestamo Prestamo::pedirPrestamo(FormularioPrestamo *form) {
    return Prestamo(form->getCuantia(), form->getCuotas(), form->getInteres());
}

File: DS_P1_S3/Dart/p1_s3/lib/prestamo.dart
//import 'Direction.dart';
//import 'MapSite.dart';

import 'FormularioPrestamo.dart';

class Prestamo{
  int num_cuotas;
  double importe;
  double interes;

  Prestamo(double imp, int cuotas, double tipo){
    importe = imp;
    num_cuotas = cuotas;
    interes = tipo;
  }

  Prestamo.otroConstructor(){

  }

  Prestamo pedirPrestamo(FormularioPrestamo f){
    Prestamo p = new Prestamo(f.cuantia,f.cuotas,f.interes);
    return p;
  }

  @override
  String toString() {
    // TODO: implement toString
    return super.toString();
  }

}